search/search.model.lkml

```diff
@@ -83,7 +83,6 @@ explore: business_development_core_search_users_monthly {
 }
 
 explore: +desktop_search_counts {
```

Comment:
Could be a good opportunity to separate the explores and their aggregate awareness LookML into their own separate files in the [Explores subfolder](https://github.com/mozilla/looker-spoke-default/tree/main/search/explores).